defaults.go

```diff
@@ -11,7 +19,7 @@ var defaultConfig = config.Config{
 		Count: 1,
 		Spec: config.Machine{
 			Name:  "node%d",
-			Image: "quay.io/footloose/centos7:0.1.0",
+			Image: "quay.io/footloose/centos7:" + imageTag(version),
```

Comment:
Hmm, I think this either should use `strconv.Quote` or sanities this input somehow. People could mistype a tag or something which could have unintended effects here.

Reply:
The tag comes from the `version` global variable, the input is really the linker option at compilation time, which we don't expose at all. The only input right now is the version set by `goreleaser`. This doesn't strike me as an issue tbh.

Reply:
Ahh okay, cool. I didn't see where it was coming from. :)